### DISCUSSION TITLE
当有多个 config 的时候，如何在多个 config 间共享通用配置？

### DISCUSSION THREAD

**Discussion by ingjieye** [MetaCubeX/mihomo#1678] [Q&A]
mihomo 用的 yaml 不支持 include 其他文件，文档里也没看到有什么办法

**Comment by mqcycy** [comment]
> mihomo 用的 yaml 不支持 include 其他文件，文档里也没看到有什么办法


必要配置覆写插件配置，非必要且需要保留的信息添加到覆写文件里，就可以随意切换yaml了
